src/components/Jobs/JobDetails/index.tsx

```diff
@@ -426,13 +434,12 @@ const JobDetailsEdit: FC<IJobDetailsEdit> = ({
     setShowCreate(false);
     setShowView(false);
   };
-
   return (
     <>
       <span className={styles['inline-job-title']}>
         <div className={styles['inline-job-div']}>
           <input
-            placeholder="Job Title here"
+            placeholder={jobEntryData.title ?? jobListingData.title}
```

Comment:
What's the diff between jobEntryData and jobListingData

Reply:
jobEntryData is old task data and on editing it becomes jobListingData

Reply:
okay